src/components/KTextArea/KTextArea.vue

```diff
@@ -224,6 +231,10 @@ export default defineComponent({
     font-family: var(--font-family-sans);
     resize: none;
 
+    &.is-resizable {
+      resize: both;
```

Comment:
question: should we change this property to `resize: vertical` so that the user cannot resize it beyond the width of its container?

Reply:
![image](https://user-images.githubusercontent.com/2229946/215598942-db7ee2d4-5d37-46da-93ce-a01a49e592cd.png)
(Otherwise it lets me do this)